src/main/java/software/amazon/awssdk/crt/CRT.java

```diff
@@ -54,6 +58,9 @@ public final class CRT {
         {
             public void run()
             {
+                ClientBootstrap.closeStaticDefault();
```

Comment:
is it safe to do in this order? requests can still come after this method and will result in those resources being recreated

Reply:
Thanks, updated it to after JVM shutdown.